app/classes/Transvision/ShowResults.php

```diff
@@ -58,6 +58,8 @@ public static function getSuggestionsResults($source_strings, $target_strings, $
         ];
         $flat_output = [];
 
+        $max_results = $max_results <= 0 ? 10 : $max_results;
```

Comment:
Approach is definitely correct. Personally I would swap the check to make it more 'natural' (we want to consider the value provided only if its bigger than 0)
```PHP
$max_results = $max_results > 0 ? $max_results : 10;
```